google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/BulkMutation.java

```diff
@@ -69,6 +69,9 @@ private void writeObject(ObjectOutputStream output) throws IOException {
   public BulkMutation add(@Nonnull String rowKey, @Nonnull Mutation mutation) {
     Preconditions.checkNotNull(rowKey);
     Preconditions.checkNotNull(mutation);
+    Preconditions.checkArgument(
```

Comment:
do you want to also check the total mutation count across rows as well?

Reply:
Hmm I'll need to iterate through all the entries I added to BulkMutation and calculate the sum everytime I add some elements. Is it worth? I think as long as someone is using the batcher this should be handled by the batcher?

Reply:
why not keep a running sum?